spec/01-unit/08-router_spec.lua

```diff
@@ -2239,7 +2346,7 @@ describe("Router", function()
           },
           route           = {
             preserve_host = true,
-            hosts         = { "preserve.com" },
+            hosts         = { "preserve.com", "preserve.com:123" },
```

Comment:
Is this new fixture value used anywhere in the tests? It seems like the intent was to ensure the request preserved Host header does include the port, which is a valuable test to have.

Reply:
removed.  it was one of the first tests i tried, but forgot to remove when got a better test